### ISSUE TITLE
URL-rewrite not created if website is added via REST on multi store system

### ISSUE THREAD

**Issue by thomvanderboon** [magento/magento2#6256]
I have a multi store website. I use the REST interface to add and maintain the products. When you add the product via the REST interface the URL rewrite is created for the product for the first website. When you add the product to another website via REST, the URL rewrite for the "extra" website/storefront is not created.

Tested on 2.1.0 and 2.2.0-dev
### Steps to reproduce
1. Create product via REST
2. Add product to extra website via REST. f.e. In PHP add product with $sku to website with id 6:
   
   ```
    $productData = array(
       'sku'               => $sku ,
       'website_id'       => 6
    );
   
    $setHaders = array('Content-Type:application/json','Authorization:Bearer '.$token);
    $productData = json_encode(array('productWebsiteLink' => $productData));
    $requestURL = "http://shop.domain.tld/rest/V1/products/".$sku."/websites";
    echo $productData."\n\n";
    $ch = curl_init();
    curl_setopt($ch,CURLOPT_URL, $requestURL);
    curl_setopt($ch,CURLOPT_POSTFIELDS, $productData);
    curl_setopt($ch, CURLOPT_CUSTOMREQUEST, "POST");
    curl_setopt($ch, CURLOPT_HTTPHEADER, $setHaders);
    curl_setopt($ch, CURLOPT_SSL_VERIFYPEER, false);
    curl_setopt($ch, CURLOPT_RETURNTRANSFER, true);
    if (curl_exec($ch)===false) {
      echo "Curl error: " . curl_error($ch)."\n";
    } else {
      $response = curl_exec($ch) ?: "";
    }
    curl_close($ch);
   ```
### Expected result
1.  Product added to website
2. URL Rewrites for all storefronts in website created
### Actual result
1.  Product added to website
2. URL Rewrites not created

**Comment by sevos1984** [comment]
Thanks for reporting. Internal ticket created MAGETWO-57653

**Comment by magento-engcom-team** [comment]
@thomvanderboon, thank you for your report.
The issue is already fixed in develop branch, 2.2.0
